app/_locales/en/messages.json

```diff
@@ -1,14 +1,14 @@
 {
   "appName": {
-    "message": "Behave!",
+    "message": "Behave! — Offensive Comments Blocker",
```

Comment:
Totally agree that controller suites the case better, but people search for blockers (i.g. ad blockers, comments blockers, and they allow to "unblock" too). This title change meant to help the app get more organic users.